README.md

```diff
@@ -95,23 +95,18 @@ First, you'll need to install at least packages for `python3`, `python-qt4` (`qt
 Then run the below script: it'll download the code and install it:
 
 ```bash
-wget https://github.com/raelgc/scudcloud/archive/v1.1.12.tar.gz
-tar -xvf v1.1.12.tar.gz
-cd scudcloud-1.1.12
-SOURCE="scudcloud-1.1"
-INSTALL="/opt/scudcloud"
-sudo mkdir -p $INSTALL/lib
-sudo mkdir -p $INSTALL/resources
-sudo cp $SOURCE/lib/*.py $INSTALL/lib
-sudo cp $SOURCE/resources/* $INSTALL/resources
-sudo cp $SOURCE/scudcloud $INSTALL
-sudo cp $SOURCE/LICENSE $INSTALL
-sudo cp $SOURCE/VERSION $INSTALL
-sudo cp $SOURCE/scudcloud.desktop /usr/share/applications
-sudo cp $SOURCE/systray/hicolor/* /usr/share/icons/hicolor/scalable/apps
-sudo cp $SOURCE/systray/mono-dark/* /usr/share/icons/mono-dark/scalable/apps
-sudo cp $SOURCE/systray/mono-light/* /usr/share/icons/mono-light/scalable/apps
-sudo ln -sf $INSTALL/scudcloud /usr/bin/scudcloud
+wget https://github.com/raelgc/scudcloud/archive/v1.2.0.tar.gz
```

Comment:
It would be wise to eliminate these lines so you don't have to update the README every release.  Just say something like 

* Download the latest release
* Unzip it
* Change into the newly created directory
* sudo python3 setup.py install